pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunction.java

```diff
@@ -41,6 +41,7 @@
 
 
 /**
+ * CHECKSTYLE:OFF
```

Comment:
Why is this needed?

Reply:
there is `arg0` in the comments which is regex'ed from checkstyle as warning